.env.example

```
@@ -92,15 +92,15 @@ NEXT_PUBLIC_ENABLE_TWO_FACTOR=false
 ROLLBAR_SERVER_ACCESS_TOKEN=
 ROLLBAR_CLIENT_ACCESS_TOKEN=
 NEXT_PUBLIC_ROLLBAR_CLIENT_ACCESS_TOKEN=
-ROLLBAR_ENABLED=1
+ROLLBAR_ENABLED=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ROLLBAR_ENABLED key should go before the ROLLBAR_SERVER_ACCESS_TOKEN key

(UnorderedKey)